### ISSUE TITLE
CTRL+Click broken on Mac

### ISSUE THREAD

**Issue by koboldunderlord** [Craiel/ArcanumScript#1]
CTRL+Click on Mac is unfortunately a right-click command.

Is it possible to pick up running on a Mac and allow the use of a different modifier key?

**Comment by Craiel** [comment]
Yes i can, unfortunatly i don't have a mac myself to test it, which shortcut would you think appropriate for mac?

**Comment by koboldunderlord** [comment]
I actually edited the script myself to support Alt instead of Ctrl!  Here's an example change:

```if (event !== undefined && event.originalEvent !== undefined && event.originalEvent.altKey === true) {```

**Comment by koboldunderlord** [comment]
Ultimately I imagine how this would function is that on window load it would determine if the user was on  Mac, and set a key variable to either `altKey` or `ctrlKey`, then the above line would be something like:


```if (event !== undefined && event.originalEvent !== undefined && event.originalEvent[repeatTriggerModifierKey] === true) {```

or something like that

**Comment by AlecZorab** [comment]
Traditionally you'd use command on a mac, rather than ctrl, so the key you're looking for is `metaKey`
